extension/docs/WebComponentsIntegrationGuide.md

```diff
@@ -576,6 +577,115 @@ Using Adyen Web Components, create `makePaymentRequest` **WITHOUT** `lineItems`
 
 </details>
 
+### Affirm payment
```

Comment:
It looks like identical so I would just update titles or description such as, our docs is long and this would make it more confusing.

`Klarna or Affirm Payment`
`For Klarna and Affirm payment methods` ...

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/785/commits/7928dda368d12d9c3f2be94bf81bfa16e046f039